packages/components/src/components/hds/layout/flex/types.ts

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
@KristinLBradley have a look at the values that I've associated to each flex prop, and let me know if you think we're missing/we should add others

Reply:
I think you've included a good range of options that seem helpful and not overly complex for consumers to understand and make use of.